mobile/src/libs/components/modal/modal.tsx

```diff
@@ -1,28 +1,64 @@
 import React from 'react';
 import { Modal as RNModal } from 'react-native';
 
-import { Icon, Pressable, View } from '#libs/components/components';
+import {
+  Button,
+  Icon,
+  Pressable,
+  Text,
+  View,
+} from '#libs/components/components';
 import { AppColor } from '#libs/enums/enums';
 
 import { styles } from './styles';
 
 type Properties = {
   isVisible: boolean;
   onClose: () => void;
+  onDelete?: () => void;
+  type?: 'Note' | 'Chat';
```

Comment:
@Liza-Veis our modal component is used for adding meditation entries also, so only when i pass onDelete it is shows functionality for deletion. and we control the deletion from outside for sure, but only pass types to show it in button title

Reply:
we can have properties like onDelete in modal, because this will mean that we want to delete a modal, also we need to remove type here because modals will have the same look. We should pass common config instead without defining for what functionality it will be used, take a look at how it's passed in react native [alert](https://reactnative.dev/docs/alert#alertbuttonstyle-ios) component